bitnami/discourse/README.md

```diff
@@ -110,7 +111,8 @@ The following table lists the configurable parameters of the Discourse chart and
 | `discourse.host`                          | Discourse host to create application URLs (include the port if =/= 80)                | `""`                                                         |
 | `discourse.siteName`                      | Discourse site name                                                                   | `My Site!`                                                   |
 | `discourse.username`                      | Admin user of the application                                                         | `user`                                                       |
-| `discourse.password`                      | Application password (min length of 10 chars)                                         | _random 10 character long alphanumeric string_               |
+| `discourse.password`                      | Application password (min length of 10 chars) - ignored if existingSecret is set to true                                        | _random 10 character long alphanumeric string_               |
```

Comment:
I think you mean "ignored if `existingSecret` is provided", am I right?

Reply:
yep :) good catch. Done